### PR TITLE
fix `Use stored `HTMLIFrameElement.contentWindow`, `HTMLIFrameElement.contentDocument` and `HTMLFrameElement.contentWindow` getters in whole code`

### PR DESCRIPTION
**Pull request by Farfurix**
https://github.com/DevExpress/testcafe-hammerhead/issues/1850

### Changes
1. Use stored `HTMLIFrameElement.contentWindow`, `HTMLIFrameElement.contentDocument` and `HTMLFrameElement.contentWindow` getters in whole code.
2. Update `eslint-plugin-hammerhead` (https://github.com/LavrovArtem/eslint-plugin-hammerhead/commit/433846fee49a765673492a0a05f95a5624a71c89)

### CODE REVIEW

File: src/client/sandbox/event/simulator.js [hunk 1]
```diff
@@ -333,8 +333,10 @@ export default class EventSimulator {
 
     _raiseNativeClick (el, originClick) {
         // NOTE: B254199
-        const curWindow       = domUtils.isElementInIframe(el) ? domUtils.getIframeByElement(el).contentWindow : window;
-        const prevWindowEvent = curWindow.event;
+        const isElementInIframe = domUtils.isElementInIframe(el);
+        const iframe            = isElementInIframe ? domUtils.getIframeByElement(el) : null;
```
Comment (LavrovArtem) [inline src/client/sandbox/event/simulator.js]:
```diff
- const isElementInIframe = domUtils.isElementInIframe(el);
- const iframe            = isElementInIframe ? domUtils.getIframeByElement(el) : null;
+ const iframe            = domUtils.isElementInIframe(el) && domUtils.getIframeByElement(el);
```

File: src/client/sandbox/event/simulator.js [hunk 2]
```diff
@@ -681,7 +683,7 @@ export default class EventSimulator {
     _raiseDispatchEvent (el, ev) {
         const isElementInIframe = domUtils.isElementInIframe(el);
         const iframe            = isElementInIframe ? domUtils.getIframeByElement(el) : null;
-        const curWindow         = iframe ? iframe.contentWindow : window;
+        const curWindow         = iframe ? nativeMethods.iframeContentWindowGetter.call(iframe) : window;
```
Comment (LavrovArtem) [inline src/client/sandbox/event/simulator.js]: the same as above

File: src/client/sandbox/iframe.js [hunk 3]
```diff
@@ -36,20 +36,25 @@ export default class IframeSandbox extends SandboxBase {
...
-            this.iframeNativeMethodsBackup.restoreDocumentMeths(iframe.contentDocument, iframe.contentWindow);
+            this.iframeNativeMethodsBackup.restoreDocumentMeths(contentDocument, contentWindow);
             this.iframeNativeMethodsBackup = null;
         }
     }
 
     _ensureIframeNativeMethodsForIE (iframe) {
-        const iframeNativeMethods = iframe.contentWindow[INTERNAL_PROPS.iframeNativeMethods];
+        const contentWindow       = nativeMethods.iframeContentWindowGetter.call(iframe);
+        const contentDocument     = nativeMethods.iframeContentDocumentGetter.call(iframe);
```
Comment (LavrovArtem) [inline src/client/sandbox/iframe.js]: Let's move it into condition

**Comment by LavrovArtem** [comment]
`+ eslint-plugin-hammerhead@0.1.11`

File: src/client/sandbox/event/simulator.js [hunk 4]
```diff
@@ -333,8 +333,9 @@ export default class EventSimulator {
 
     _raiseNativeClick (el, originClick) {
         // NOTE: B254199
-        const curWindow       = domUtils.isElementInIframe(el) ? domUtils.getIframeByElement(el).contentWindow : window;
-        const prevWindowEvent = curWindow.event;
+        const iframe            = domUtils.isElementInIframe(el) && domUtils.getIframeByElement(el);
+        const curWindow         = iframe ? nativeMethods.contentWindowGetter.call(iframe) : window;
+        const prevWindowEvent   = curWindow.event;
```
Comment (LavrovArtem) [inline src/client/sandbox/event/simulator.js]: extra whitespaces

File: src/client/sandbox/event/simulator.js [hunk 5]
```diff
@@ -679,9 +680,8 @@ export default class EventSimulator {
     }
 
     _raiseDispatchEvent (el, ev) {
-        const isElementInIframe = domUtils.isElementInIframe(el);
-        const iframe            = isElementInIframe ? domUtils.getIframeByElement(el) : null;
-        const curWindow         = iframe ? iframe.contentWindow : window;
+        const iframe            = domUtils.isElementInIframe(el) && domUtils.getIframeByElement(el);
+        const curWindow         = iframe ? nativeMethods.contentWindowGetter.call(iframe) : window;
```
Comment (LavrovArtem) [inline src/client/sandbox/event/simulator.js]: extra whitespaces

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Pull request merged**